results/cmd/api/main.go

```diff
@@ -57,7 +65,27 @@ type server struct {
 	db *sql.DB
 }
 
+// Receive TaskRun from Watcher and save it to local Sqlite Server
 func (s *server) CreateTaskRun(ctx context.Context, req *pb.CreateTaskRunRequest) (*pb.TaskRun, error) {
-	// TODO: implement this.
-	return nil, status.Errorf(codes.Unimplemented, "method CreateTaskRun not implemented")
+	database := s.db
+	statement, err := database.Prepare("INSERT INTO taskrun (taskrunlog, uid, name, namespace) VALUES (?, ?, ?, ?)")
+	if err != nil {
+		log.Fatalf("failed to insert a new taskrun: %v", err)
+	}
+
+	// serialize data and insert it into database
+	taskrunFromClient := req.GetTaskRun()
+	blobData, err := proto.Marshal(taskrunFromClient)
+	if err != nil {
+		log.Fatal("marshaling error: ", err)
```

Comment:
`log.Fatal` will exit the process..we can log the error but we should also return it e.g.
```
if err != nil {
  return nil, fmt.Errorf("failed to marshal proto: %v", err) 
```

Reply:
done